src/js/player.js

```diff
@@ -1620,30 +1618,29 @@ class Player extends Component {
    *         return undefined.
    */
   play() {
-    if (this.changingSrc_) {
-      this.ready(function() {
-        const retval = this.techGet_('play');
 
-        // silence errors (unhandled promise from play)
-        if (retval !== undefined && typeof retval.then === 'function') {
-          retval.then(null, (e) => {});
-        }
-      });
+    // Given a return value for a tech's `play()` method, silences "unhandled
+    // promise from play" errors.
+    const silencePromise = (value) => {
```

Comment:
we probably want a promise helper file that contains `isPromise` and `silencePromise`, since we silence promises in a few places.